Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -1,20 +1,40 @@
 import Foundation
 
+#if canImport(BraintreeCore)
+import BraintreeCore
+#endif
+
 /// Options for the PayPal edit funding instrument flow
 /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-public struct BTPayPalVaultEditRequest {
+public class BTPayPalVaultEditRequest {
 
+    var riskCorrelationID: String?
+    let hermesPath: String = "v1/paypal_hermes/generate_edit_fi_url"
```

Comment:
Lets just have this live at the callsite since it's not necessary to live here for the edit flow